### PR TITLE
Add cluster-storage-operator to CI

### PR DESCRIPTION
**Pull request by wongma7**

### CODE REVIEW

File: ci-operator/jobs/openshift/cluster-storage-operator/openshift-cluster-storage-operator-openshift-csi-driver-registrar-master-postsubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+postsubmits:
```
Comment (wongma7) [inline ci-operator/jobs/openshift/cluster-storage-operator/openshift-cluster-storage-operator-openshift-csi-driver-registrar-master-postsubmits.yaml]: don't know why this file exists

File: ci-operator/config/openshift/cluster-storage-operator/openshift-cluster-storage-operator-master.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    tag: golang-1.10
+tests:
+- as: unit
+  commands: make test
+  container:
+    from: src
+- as: verify
+  commands: make verify
+  container:
+    from: src
```
Comment (smarterclayton) [inline ci-operator/config/openshift/cluster-storage-operator/openshift-cluster-storage-operator-master.yaml]: you need the aws e2e job here

Comment (wongma7) [reply ci-operator/config/openshift/cluster-storage-operator/openshift-cluster-storage-operator-master.yaml]: okay, added. does this `Secrets` test focus I've copied run after my image has been built & manifests applied to the cluster? Or before, as some kind of sanity check?

**Comment by stevekuznetsov** [comment]
@smarterclayton PTAL

File: ci-operator/config/openshift/cluster-storage-operator/openshift-cluster-storage-operator-master.yaml (hunk 2 above)
Comment (bbguimaraes) [reply ci-operator/config/openshift/cluster-storage-operator/openshift-cluster-storage-operator-master.yaml]: @wongma7: this test type (`openshift_installer`) runs the installer to deploy a cluster with the release images + the images that were built from the PR, then runs the ginkgo test(s).

**Comment by smarterclayton** [comment]
Before you merge this, 

https://github.com/openshift/cluster-storage-operator/blob/master/Dockerfile

needs to look like:

https://github.com/openshift/cluster-ingress-operator/blob/master/Dockerfile

The only things that should be different are your custom build command, your directory, and the binary you copy.  Please copy the formatting as well, we're looking to enforce style very closely with automation in the future.  You can see how other operators are in a specific pattern as well

https://github.com/openshift/cluster-dns-operator/blob/master/Dockerfile

/hold

**Comment by smarterclayton** [comment]
No alpine images should be in use anywhere in our chain.  Everyone must descend from base.

**Comment by wongma7** [comment]
@smarterclayton fixed the Dockerfile https://github.com/openshift/cluster-storage-operator/blob/master/Dockerfile

**Comment by smarterclayton** [comment]
/lgtm

thanks

**Pull request merged**